logging/logging.go

```diff
@@ -687,6 +687,15 @@ type HTTPRequest struct {
 	// validated with the origin server before being served from cache. This
 	// field is only meaningful if CacheHit is true.
 	CacheValidatedWithOriginServer bool
+
+	// The number of HTTP response bytes inserted into cache. Set only when a cache fill was attempted.
+	CacheFillBytes int64
+
+	// Whether or not a cache lookup was attempted.
+	CacheLookup bool
+
+	// Protocol used for the request. Examples: "HTTP/1.1", "HTTP/2", "websocket"
+	Protocol string
```

Comment:
Why do you need this field, if below r.Request.Proto is used instead?

Reply:
There are two httpRequest objects here. One is based on Protobuf definition(this one) and another one is Go's http request. There is translation between the two. Proto-based one has Protocol as one of the fields in proto.

Reply:
We have even 3 http requests.

1. HttpRequest Proto that is defined in google.golang.org/genproto/googleapis/logging/type and is send to Cloud Logging.
2. http.Request - standard struct from go library.
3. HTTPRequest - defined in this library, contains instance of http.Request (2), plus adds few more fields that are missing from http.Request (2), but required for HttpRequest proto (1). 

We have an option: either provide a new Protocol field to HTTPRequest (3), but then below it should be something like:
Protocol: r.Protocol

or we rely just on Proto field of http.Request (2) as it's done right now:
Protocol: r.Request.Proto

Reply:
I prefer the second option

Reply:
In such case Protocol field is no needed here (you can try to remove it and see that it still builds)

Reply:
removed the field. Thanks!